README.md

```diff
@@ -1,14 +1,16 @@
-# README
```

Comment:
While reviewing the PR, I realised that I need to find some code examples every time. I think it's a good idea to create a `code-examples` folder then put the examples that we used in the screenshots, ie. example.py, example.java... so that if anything changes, we can go and double check, then generate new screenshots, etc...

What do you think? I'm asking since you're probably having the new languages somewhere and can quickly put them in `code-examples` folder. I'll try and make the files for old languages in another PR.

Reply:
yup, this is a good idea. I'll had my examples